packages/expo/package.json

```diff
@@ -155,6 +155,7 @@
     "@types/react-test-renderer": "^16.8.1",
     "expo-module-scripts": "^1.0.0",
     "react": "16.8.3",
+    "react-dom": "^16.8.6",
```

Comment:
Hmm, since we use exact version of `react` package, shouldn't we use exact version here? Or maybe `~` at least 🤔

Reply:
react-dom can use `^`. It's just the interface between react and react-native that does not follow semver (using the wrong patch version of react will break react-native in some cases).